spec/rspec/core/formatters/base_text_formatter_spec.rb

```diff
@@ -30,33 +30,48 @@
       expect(output.string).to match("2 examples, 2 failures, 2 pending")
     end
 
-    it "includes command to re-run each failed example" do
-      example_group = RSpec.describe("example group") do
-        it("fails") { fail }
+    describe "rerun command for failed examples" do
+      it "uses the location to identify the example" do
+        example_group = RSpec.describe("example group") do
+          it("fails") { fail }
+        end
+        line = __LINE__ - 2
```

Comment:
whilst this is not a change made by this pull request, it is weird to me that the line number used is after the example here and before the example in the next spec, can we unify them?